docs/Learning-Environment-Create-New.md

```diff
@@ -435,45 +434,22 @@ well the Agent is achieving the task. In this example, the maximum reward an
 Agent can earn is 1.0, so these statistics approach that value when the Agent
 has successfully *solved* the problem.
 
-**Note:** If you use TensorBoard, always increment or change the `run-id`
-you pass to the `mlagents-learn` command for each training run. If you use
-the same id value, the statistics for multiple runs are combined and become
-difficult to interpret.
-
 ## Optional: Multiple Training Areas within the Same Scene
 
-In many of the [example environments](Learning-Environment-Examples.md), many copies of
-the training area are instantiated in the scene. This generally speeds up training,
-allowing the environment to gather many experiences in parallel. This can be achieved
-simply by instantiating many Agents which share the `Behavior Parameters`. Use the following steps to
-parallelize your RollerBall environment.
+In many of the [example environments](Learning-Environment-Examples.md), many
+copies of the training area are instantiated in the scene. This generally speeds
+up training, allowing the environment to gather many experiences in parallel.
+This can be achieved simply by instantiating many Agents which share the
```

Comment:
> which share the `Behavior Parameters`

This isn't really accurate; they should have identical `Behavior Parameters` but they're not actually "shared" .

Reply:
Good catch - fixed it.